validator/specs/lib/validateTilesetSpec.js

```diff
@@ -81,18 +81,18 @@ describe('validateTileset', function() {
 
 
 describe('validateBoundingVolume', function() {
-   var regionTileset = {
+    var regionTileset = {
        "root": {
            "boundingVolume": {
-               "region": [10, 10, 10, 10, 10, 88]
+               "region": [20, 40, 50, 10, 10, 88]
            },
            "content": {
                "boundingVolume": {
-                   "region":  [0, 0, 0, 0, 20, 30]
-               },
+                   "region":  [18, 14, 15, 8, 20, 30]
```

Comment:
> content: [18, 14, 15, 8] and tile: [20, 40, 50, 10]

Based on the layout: [west, south, east, north]

* Within the content region or tile region, west should be less than east, and south should be less north.
* The content west/east don't fit within the tile's west/east
* The content south/north also don't fit.